RecoLocalCalo/HcalRecProducers/src/HcalRecHitSoAToLegacy.cc

```diff
@@ -0,0 +1,72 @@
+#include <iostream>
+#include <string>
+
+#include "DataFormats/HcalRecHit/interface/HcalRecHitHostCollection.h"
+#include "DataFormats/HcalRecHit/interface/HcalRecHitCollections.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/EDPutToken.h"
+
+class HcalRecHitSoAToLegacy : public edm::stream::EDProducer<> {
+public:
+  explicit HcalRecHitSoAToLegacy(edm::ParameterSet const& ps);
+  ~HcalRecHitSoAToLegacy() override = default;
+
+  static void fillDescriptions(edm::ConfigurationDescriptions&);
+
+private:
+  void produce(edm::Event&, edm::EventSetup const&) override;
+
+private:
+  using IProductType = hcal::RecHitHostCollection;
+  const edm::EDGetTokenT<IProductType> recHitsM0TokenIn_;
```

Comment:
```suggestion
  const edm::EDGetTokenT<hcal::RecHitHostCollection> recHitsTokenIn_;
```

Reply:
Updated in 715105876f8999402d3e30f2fa94579275479978